layouts/blog/post.html

```diff
@@ -20,6 +20,33 @@ <h1>{{ .Title }}</h1>
             </figure>
             {{ end }}
             {{ end }}
+
+            <h4>Share this blog</h4>
+
+            <!-- Sharingbutton Facebook -->
+            <a class="resp-sharing-button__link" href="https://facebook.com/sharer/sharer.php?u={{ .Permalink }}" target="_blank" rel="noopener" aria-label="">
```

Comment:
I need to test this, but I think we could use the `aria-label` here to help folk with sight impairment. Same goes for other buttons.

```suggestion
            <a class="resp-sharing-button__link" href="https://facebook.com/sharer/sharer.php?u={{ .Permalink }}" target="_blank" rel="noopener" aria-label="Share on Facebook">
```

Reply:
Agreed. I'll add them.